db/etl/create_us_sql.py

```
@@ -0,0 +1,153 @@
+                f.write(city + ",\n")
+            else:
+                # Last city doesn't need a comma
+                f.write(city + ";\n")
+
+    print(f"US cities SQL file created at {output_path}")
+    return True
+
+
+def main():
+    # Download world.sql if it doesn't exist
+    world_sql_path = download_sql_file()
```

Ruff (0.8.2)

Local variable `world_sql_path` is assigned to but never used

Remove assignment to unused variable `world_sql_path`

(F841)